lib/ts/recipe/session/utils.ts

```diff
@@ -207,7 +207,7 @@ export function validateAndNormaliseUserInput(
     let accessTokenPayloadJWTPropertyName = "jwt";
     let issuer: string | undefined;
 
-    if (config !== undefined && config.jwt !== undefined && config.jwt.enable === true) {
+    if (config !== undefined && config.jwt !== undefined && config.jwt.enable) {
```

Comment:
please revert this change as it's not related to this PR

Reply:
Ok, will do